.claude/skills/wikitextparser/skills/01-wikitext-basics.md

```
@@ -0,0 +1,250 @@
+
+`remove_markup(s, **kwargs)` is exactly equivalent to
+`wtp.parse(s).plain_text(**kwargs, _is_root_node=True)`.
+
+### 4. Pretty-print templates
+
+```python
+parsed = wtp.parse("{{Infobox|name=Alice|age=30|nationality=US}}")
+print(parsed.pformat())
+```
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)